app/views/transactions/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_with(model: @transaction, url: submit_path, method: method, class: 'd-flex flex-column w-100') do |f| %>
+  <div class="d-flex align-items-center justify-content-between bg-primary px-3 py-2 text-white">
+    <div><%= link_to backpath, class: "text-white" do %> <i class="fa fa-arrow-left" aria-hidden="true"> <% end %></i></div><h4>New Transaction</h4><div><%= f.submit "Next", class: 'btn text-right px-0 text-white' %></div>
+  </div>
+  <% if @transaction.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(@transaction.errors.count, "error") %> prohibited this transaction from being saved:</h2>
+
+      <ul>
+        <% @transaction.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <%= f.text_field :name, autofocus: true, autocomplete: 'new-transaction', placeholder: 'Ex: Electricity', skip_label: true, class: 'mt-2 border-right-0 border-top-0 border-left-0 border-bottom form-control shadow-none' %>
+  <%= f.number_field :amount, autofocus: true, autocomplete: 'new-icon-url', placeholder: 'Ex: 250', skip_label: true, class: 'mt-2 border-right-0 border-top-0 border-left-0 border-bottom form-control shadow-none' %>
+  <%= f.submit "Submit", class: 'btn px-3 bg-success py-3 mt-auto position-fixed text-center text-white add-edit-button' %>
+<% end %>
```

Comment:
- [ ] According to the project requirement, when creating a transaction, the category for which a transaction is created should be selected from this view. Kindly show a select box to display the category/categories from which one can select on which a transaction is created. 

<img width="1036" alt="Screenshot 2023-07-08 at 11 15 24 AM" src="https://github.com/veronica365/budget-buddy/assets/50941074/6f5de60b-4bc4-4b95-9d81-58a4fb5d3385">